methods/dst.py

```diff
@@ -29,6 +28,10 @@
 LOGO_FINDER = 2
 
 
+# Thread worker instance shared for different concurrent parts
+worker = ThreadWorker()
```

Comment:
currently only used for SAN domain checks, maybe rename and use preprocessor for it?